### PR TITLE
updated merge-sort

### PR DESCRIPTION
**Pull request by jainaman224**
#1048

### CODE REVIEW

File: Algorithms-Merge-Sort.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Most modern languages have an inbuilt sort() function which automatically sorts an input array or list. Did you ever wonder how the sort function actually works on the inside?. Knowing common sorting algorithms and their implementations is the most important part of a coding interview. In this series of articles, we will look at several important sorting algorithms. How they are implemented, the time and space complexity etc. Our very first post is on Merge Sort.
 
-##  
+##
```
Comment (Rafase282) [inline Algorithms-Merge-Sort.md]: we need the two spaces next to `##`

**Comment by atjonathan** [comment]
### **LGTM :+1:**

File: Algorithms-Merge-Sort.md [hunk 2]
```diff
@@ -13,36 +13,124 @@ To learn about Merge Sort, a basic knowledge about [Recursion](http://programmer
 The biggest advantage of using Merge sort is that the [time complexity](https://www.youtube.com/watch?v=V42FBiohc6c&list=PL2_aWCzGMAwI9HK8YPVBjElbLbI3ufctn) is only n*log(n) to sort an entire Array. It is a lot better than n^2 running time of bubble sort or insertion sort.
 Before we write the JavaScript code, let us understand how merge sort works with the help of a diagram.
 
-![alt tag] (https://i67.tinypic.com/2ahe49y.png)
+![Merge Sort] (https://i67.tinypic.com/2ahe49y.png)
```
Comment (Rafase282) [inline Algorithms-Merge-Sort.md]:
```diff
-![Merge Sort] (https://i67.tinypic.com/2ahe49y.png)
+![Merge Sort](https://i67.tinypic.com/2ahe49y.png)
```

File: Algorithms-Merge-Sort.md [hunk 3]
```diff
@@ -13,36 +13,124 @@ To learn about Merge Sort, a basic knowledge about [Recursion](http://programmer
 The biggest advantage of using Merge sort is that the [time complexity](https://www.youtube.com/watch?v=V42FBiohc6c&list=PL2_aWCzGMAwI9HK8YPVBjElbLbI3ufctn) is only n*log(n) to sort an entire Array. It is a lot better than n^2 running time of bubble sort or insertion sort.
 Before we write the JavaScript code, let us understand how merge sort works with the help of a diagram.
 
-![alt tag] (https://i67.tinypic.com/2ahe49y.png)
+![Merge Sort](https://i67.tinypic.com/2ahe49y.png)
```
Comment (Rafase282) [inline Algorithms-Merge-Sort.md]: Please change the link to https://i.imgur.com/ac98ccu.png

That link is not working, I uploaded the image to the one I just posted.

**Comment by Rafase282** [comment]
Thanks @jainaman224

**Pull request merged**